plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -371,6 +371,7 @@ public static function register_post_types() {
 							array(
 								'id'    => 'general',
 								'title' => __( 'General', 'woocommerce' ),
+								'order' => 10,
```

Comment:
Just noting that I think this is a good short-term fix, but long-term I'd love to see this `order` property removed since this array already determines order and may conflict with the number provided here.

Do we have another issue already to track this long-term and provide a fix for the underlying cause?

Reply:
Originally I believe both @louwie17  and I were considering this a potential future extensibility point, in which case you'd want a way to control the order by using the slot-fill directly. Now that you mention it, though, that wouldn't ultimately make sense since you'd use to use the block extensibility hooks to relay the inner blocks anyways. 

If we're considering this exclusive to internal use then I'd agree that the usage of the `order` is just a temporary measure. Louren's created an issue with GB [here](https://github.com/WordPress/gutenberg/issues/50163), but the only feedback so far doesn't consider it a bug. We may want to weigh in there further.

Reply:
It actually looks like this may [have just been resolved upstream](https://github.com/WordPress/gutenberg/pull/50292).

Reply:
Thanks for updating the future concerns doc, we should check if that does solve our problem, maybe when Gutenberg  `15.8` rolls out? As I am not sure how to best build and run Gutenbergs dev version (although I am sure its relatively straightforward).
But the basic ordering may still be needed as this won't be available until the follow WP version.

Reply:
I just tested this with the latest GB trunk version and it does seem to be fixed now 🎉